_posts/2020-06-09-changelog-124.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #124"
+date: 2020-06-15 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5514 -->
+* Support 2020.2 EAPs
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5496 -->
+* Enable [new macro expansion engine](https://www.jetbrains.com/help/clion/rust-support.html#language-support) in nightly builds
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5181 -->
+
+* Provide initial version of [`Move`](https://www.jetbrains.com/help/idea/move-refactorings.html) (<kbd>F6</kbd> or drag&drop) refactoring.
+Currently, only supports moving a file to same crate if moved file has no child modules. In other cases, no references will be updated
+
+    {% include gif-img.html path="/assets/posts/changelog-124/5181" w="700px" %}
+
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/4460 -->
+* Provide initial version of [`Inline method`](https://www.jetbrains.com/help/idea/inline.html) (<kbd>Ctrl+Alt+N</kbd>) refactoring.
+Note, actual arguments are not inlined properly so far (by [@SaarYogev])
+
+    {% include gif-img.html path="/assets/posts/changelog-124/4460" w="700px" %}
+
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5430 -->
+* Add completion of existing file paths inside path-like string literals (by [@Kobzol])
```

Comment:
Let's add a note, that it works only when literal is direct argument of method/function call because in other languages (Java and Python, for example) it also may work for local variables

Reply:
`Add completion of existing file paths inside path-like string literals in function arguments`